examples/Generic/README.md

```diff
@@ -2,21 +2,160 @@
 
 This directory contains a minimal example that shows how to use `AstarteGenericDevice`.
 
-The device is instantiated in `ExampleDevice.java`, the code is commented to highlight all the
-things needeed to create your Astarte Device.
+The device is instantiated in `ExampleDevice`, the code is commented to highlight all the
+things needed to create your Astarte Device.
+
+
+## Downloading
+
+Add to "dependencies" (as for any module you need):
+
+``` groovy
+dependencies {
+    implementation 'org.astarte-platform:devicesdk-generic:$version'
+}
+```
+
+Usage
```

Comment:
I would avoid mixing `#` and `===`/`---` tiles styles. I would stick to `#` style.